mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/ConfirmCompositeSwapScreenV2.tsx

```diff
@@ -160,245 +216,320 @@ export function ConfirmCompositeSwapScreenV2({ route }: Props): JSX.Element {
+                  },
+                }}
+                rhs={{
+                  value: futureSwap.executionBlock,
+                  testID: "confirm_text_transaction_date",
+                }}
+              />
+              <TextRowV2
+                lhs={{
+                  value: "",
+                  testID: "",
+                }}
+                rhs={{
+                  value: futureSwap.transactionDate,
+                  testID: "confirm_text_transaction_date",
+                }}
+              />
+            </View>
+            <ThemedViewV2
+              dark={tailwind("border-gray-700")}
+              light={tailwind("border-gray-300")}
+              style={tailwind("py-5 border-b-0.5")}
+            >
+              <TextRowV2
+                lhs={{
+                  value: translate(
+                    "screens/ConfirmCompositeSwapScreen",
+                    "To receive (est.)"
+                  ),
+                  testID: "settlement_block",
+                  themedProps: {
+                    light: tailwind("text-mono-light-v2-500"),
+                    dark: tailwind("text-mono-dark-v2-500"),
+                  },
+                }}
+                rhs={{
+                  value: `${tokenB.displaySymbol}`,
+                  testID: "confirm_text_transaction_date",
+                  themedProps: {
+                    light: tailwind("text-mono-light-v2-900"),
+                    dark: tailwind("text-mono-dark-v2-900"),
+                  },
+                }}
+              />
+              <TextRowV2
+                lhs={{
+                  value: "",
+                  testID: "",
+                }}
+                rhs={{
+                  value: translate(
+                    "screens/CompositeSwapScreen",
+                    "Oracle price {{percentageChange}}",
+                    {
+                      percentageChange: futureSwap.oraclePriceText,
+                    }
+                  ),
+                  testID: "confirm_estimated_to_receive",
+                }}
+              />
+            </ThemedViewV2>
+          </>
+        ) : (
+          <ThemedViewV2
+            dark={tailwind("border-gray-700")}
+            light={tailwind("border-gray-300")}
+            style={tailwind("py-5 border-b-0.5")}
+          >
+            <NumberRowV2
+              lhs={{
+                value: translate(
+                  "screens/ConfirmCompositeSwapScreen",
+                  "To receive (incl. of fees"
+                ), // estimated return less dex fees
+                testID: "estimated_to_receive",
+                themedProps: {
+                  light: tailwind("text-mono-light-v2-500"),
+                  dark: tailwind("text-mono-dark-v2-500"),
+                },
+              }}
+              rhs={{
+                testID: "confirm_estimated_to_receive",
+                value: swap.amountTo.toFixed(8),
```

Comment:
This should be the `estimatedReturnLessDexFees`
Why?
1. The transaction fee is out of the swap transaction. It is taken out of the reserved DFI
2. It is already in tokenB unit and inclusive of commission and dex fees